### PR TITLE
Prestidigitation bundle

### PR DESCRIPTION
**Pull request by MadmanMartian**
Adds a bundle that grants an assortment of spells
 - Extinguish: Extinguish the target
 - Sparks: Create sparks at the given location
 - Clean: Cleans the targeted object
 - Dirty: Drops the targeted object into the realm of blood for a moment, covering it in ichor
 - Summon mundane trinket: Creates a coin for 30 seconds
 - Warm object: Warms the targeted object
 - Cool object: Cools the targeted object

:cl:
 * rscadd: Adds the prestidigitation bundle. This gives some misc spells that have minor effects.

### CODE REVIEW

**Comment by Shadowmech88** [comment]
A minor conjuration spell that let you spawn a mundane but potentially useful tool out of a large whitelist could be useful.
Not necessarily for this PR, though.

**Comment by MadmanMartian** [comment]
Reach into bluespace toolbox

Could be any tool, lasts 15 seconds

**Comment by sonime** [comment]
Now you've done it.
Where's Thaumaturgy
Where's Shillelagh
Where's Vicious Mockery 
Where's Eldritch Blast
That's just cantrips, I can go on.

**Comment by Pathid** [comment]
More Prestidigitation effects that seem workable:
* Distant music: Functions like a faint musical instrument.
* Inscribe: Combination crayon and chisel that works at a distance. What it writes is temporary.
* Gust: Blow some small items in a direction.

**Comment by spoonylove** [comment]
Incoming wiz janitor

**Comment by SonixApache** [comment]
fire bolt

File: code/modules/spells/targeted/prestidigitation.dm [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+	level_max = list()
+	hud_state = "coin"
+
+/spell/targeted/create_trinket/cast(var/list/targets, mob/user)
+	var/static/list/available_trinkets = existing_typesof(/obj/item/weapon/coin)-list(/obj/item/weapon/coin/trader, /obj/item/weapon/coin/adamantine)
+	var/choice = pick(available_trinkets)
+	var/obj/item/I = new choice
+	user.put_in_hands(I)
+	spawn(300)
+		if(istype(I.loc, /mob/living))
```
Comment (DamianX) [inline code/modules/spells/targeted/prestidigitation.dm:77]: dropping should be handled by qdel already

File: code/modules/spells/targeted/prestidigitation.dm [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+	spell_flags = INCLUDEUSER
+	level_max = list()
+	hud_state = "coin"
+
+/spell/targeted/create_trinket/cast(var/list/targets, mob/user)
+	var/static/list/available_trinkets = existing_typesof(/obj/item/weapon/coin)-list(/obj/item/weapon/coin/trader, /obj/item/weapon/coin/adamantine)
+	var/choice = pick(available_trinkets)
+	var/obj/item/I = new choice
+	user.put_in_hands(I)
+	spawn(300)
```
Comment (DamianX) [inline code/modules/spells/targeted/prestidigitation.dm:76]: the SECONDS macro is a thing

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
Does warm/cool do anything at all?

**Comment by MadmanMartian** [comment]
Can be used for chemistry, and assuming that spicy keychain pr is done properly, that.

**Comment by MadmanMartian** [comment]
Oh, and blacksmithing

**Comment by Dacendeth** [comment]
would warm/cooling work for the shard?

**Comment by Pathid** [comment]
@Dacendeth Nope.

**Comment by sonime** [comment]
I retract what I said earlier. Most of those spells aren't a part of the Wizard's already dumb spell list.
Instead, let me talk about wizards.

Where's Shield
Where's Mage Armor
Where's Counterspell
Where's Acid Splash
Where's Tasha's Hideous Laughter
Where's Comprehend Languages
Where's literally any Divination spell
Where's Suggestion
Where's Nondetection
Where's Mage Hand

Fuck wizards, fuck you, and fuck me too.

**Comment by SonixApache** [comment]
- trinket would let you get the wizarditis bottle from the magivend
 - why isn't this all just one spell

also conflicts

**Comment by MadmanMartian** [comment]
@SonixApache 
 - Neat little feature I guess
 - Because having a menu for all that would be a pain

**Comment by SonixApache** [comment]
I think having one menu for a non-combat trick spell with various effects would be better than cluttering the spell bar / list with 7 (seven) more spells.

**Comment by DamianX** [comment]
conflics

**Review by Intigracy — DISMISSED** [review]
confrictu

**Comment by Blithering** [comment]
Teleporting someone into the blood dimension should make them come back out screaming

**Pull request merged**